activation/activation_test.go

```diff
@@ -213,9 +215,12 @@ func newBuilder(activationDb ATXDBProvider) *Builder {
 func setActivesetSizeInCache(t *testing.T, activesetSize uint32) {
 	view, err := meshProvider.GetOrphanBlocksBefore(meshProvider.LatestLayer())
 	assert.NoError(t, err)
+	sort.Slice(view, func(i, j int) bool {
```

Comment:
use your function you wrote in calculating atx view hash to get layer hash

Reply:
As discussed, since this is a test, I'll leave it as is.